lib/json_and_string_parameter_filter.rb

```diff
@@ -40,10 +40,10 @@ def run(param_key, param_value)
       # no need to do anything with param_value as it's nil
     elsif param_value.is_a?(String)
       if param_key =~ @string_regexp
-        param_value.gsub!(/^.*$/, '[FILTERED]')
+        param_value.gsub!(/^.*$/, '[FILTERED]').encode('UTF-8', 'binary', invalid: :replace, undef: :replace, replace: '')
       else
         @value_filters.each do |value_filter|
-          param_value.gsub!(/^.*$/, '[FILTERED]') if value_filter.call(param_value)
+          param_value.gsub!(/^.*$/, '[FILTERED]').encode('UTF-8', 'binary', invalid: :replace, undef: :replace, replace: '') if value_filter.call(param_value.encode('UTF-8', 'binary', invalid: :replace, undef: :replace, replace: ''))
```

Comment:
I would probably put `param_value.gsub!(/^.*$/, '[FILTERED]').encode('UTF-8', 'binary', invalid: :replace, undef: :replace, replace: '')` in some variable just to avoid writing the same operation 3 times.